setup.py

```diff
@@ -50,6 +50,7 @@
         'Flask-CORS>=2.1.0',
     ],
     'docs': [
+        'dicttoxml>=1.7.4',
```

Comment:
You can add it as a test dependency since it is the example app which requires it and not only the documentation.